test/IRGen/cf_objc_retainAutoreleasedReturnValue.swift

```diff
@@ -0,0 +1,20 @@
+// RUN: %target-swift-frontend -module-name cf_objc_retainAutoreleasedReturnValue -I %S/Inputs %s -enable-objc-interop -emit-ir | %FileCheck %s
+
+// We need to require objc_codegen to avoid this test on WASM.
+// (That's why the other half of this test is in a separate file.)
```

Comment:
The other half side is forgotten to be included in the commit?

Reply:
Oops, quite right.